src/components/policy/policy_regular/src/cache_manager.cc

```diff
@@ -872,6 +872,53 @@ const boost::optional<bool> CacheManager::LockScreenDismissalEnabledState()
   return empty;
 }
 
+const boost::optional<std::string>
+CacheManager::LockScreenDismissalWarningMessage(
```

Comment:
@IGapchuk double check of common implementation

Reply:
@mrapitis We can make common implementation in separate entity, but will be affected whole Cache Manager class. Because in this case we need to move into separate entity more of functionality implementation, not only  LockScreenDismissalWarningMessage method. We believe, that we shouldn't do that in scope of this feature.
This changes is rather big refactoring that is always hard to push throw project maintainers.